src/components/user/EditProfile.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { object, func } from 'prop-types';
+
+import SmilesLogo from 'resources/icons/SmilesLogo';
+
+import EditProfileForm from 'components/user/EditProfileForm';
+import MenuHeader from 'components/common/MenuHeader';
+import { updateUser } from 'actions/userActions';
+
+const EditProfile = ({ history, updateUser }) => (
+  <div className="page-container page-container--full-height overlap-menu">
+    <MenuHeader title="Edit Profile" goBack={history.goBack} />
```

Comment:
I see the
```
<MenuHeader .... />
... something ...
<div className="footer show-for-medium">
  <SmilesLogo className="logo--small" />
</div>
```
pattern repeated many times. What about making it a new, reusable component where you wrap children?